docs/usage/files.md

```diff
@@ -171,6 +171,30 @@ For large files or in cases where the network connection is less reliable,
 you may want to upload the file in parts.  This allows a single part to fail
 without aborting the entire upload, and failed parts can then be retried.
 
+## Automatic Uploader
+
+The SDK provides a method of automatically handling a chunked upload; simply call [`chunked_upload.start()`][start] with
+the path to the path you wish to upload method once you have constructed an [`upload_session`][upload_session] from the
+[`File`][file_class] you wish to upload a new version of or for a new file upload construct an [`upload_session`][upload_session]
+from the [`Folder`][folder_class].
```

Comment:
This is missing a description of the method signature and link to readthedocs for the `Folder` and `File` methods — it would be helpful to include those for reference